network-api/networkapi/buyersguide/factory.py

```diff
@@ -112,50 +99,148 @@ def product_category(self, create, extracted, **kwargs):
             else:
                 return
 
+    blurb = Faker('sentence')
+    url = Faker('url')
+    price = LazyAttribute(lambda _: randint(49, 1500))
+
+    @post_generation
+    def set_image(self, create, extracted, **kwargs):
+        if settings.USE_CLOUDINARY:
+            self.cloudinary_image = Faker('product_image').generate({})
+        else:
+            self.image.name = Faker('product_image').generate({})
+
+    worst_case = Faker('sentence')
+
+    signup_requires_email = LazyFunction(get_extended_yes_no_value)
+    signup_requires_phone = LazyFunction(get_extended_yes_no_value)
+    signup_requires_third_party_account = LazyFunction(get_extended_yes_no_value)
+    signup_requirement_explanation = Faker('sentence')
+
+    how_does_it_use_data_collected = Faker('sentence')
+    data_collection_policy_is_bad = Faker('boolean')
+    user_friendly_privacy_policy = LazyFunction(get_extended_yes_no_value)
+
+    meets_minimum_security_standards = Faker('boolean')
+    show_ding_for_minimum_security_standards = Faker('boolean')
+    uses_encryption = LazyFunction(get_extended_yes_no_value)
+    uses_encryption_helptext = Faker('sentence')
+    security_updates = LazyFunction(get_extended_yes_no_value)
+    security_updates_helptext = Faker('sentence')
+    strong_password = LazyFunction(get_extended_yes_no_value)
+    strong_password_helptext = Faker('sentence')
+    manage_vulnerabilities = LazyFunction(get_extended_yes_no_value)
+    manage_vulnerabilities_helptext = Faker('sentence')
+    privacy_policy = LazyFunction(get_extended_yes_no_value)
+    privacy_policy_helptext = Faker('sentence')
+
     @post_generation
     def set_privacy_policy_link(self, create, extracted, **kwargs):
         ProductPrivacyPolicyLinkFactory.create(product=self)
 
+    phone_number = Faker('phone_number')
+    live_chat = Faker('url')
+    email = Faker('email')
+
+    if random() > 0.5:
+        twitter = '@TwitterHandle',
+
+    # updates...
```

Comment:
Is this missing anything here?

Reply:
probably. Updates are factory'd independently, but this is where you'd "expect" them, so it's a comment that should either go, or be refined, but not in this PR.